packages/dev/test/bls/BatchFlow.test.ts

```diff
@@ -77,21 +81,41 @@ contract.only('Batch Relaying Flow', function (accounts: string[]) {
     // @ts-ignore
     TestToken.web3.setProvider(bathingRelayProvider, undefined)
     testToken = await TestToken.at(env.testToken.address)
-    await env.testToken.mint(200000, {
+    await env.testToken.mint(1000000, {
       from: accounts[0]
     })
+    await env.testToken.mint(1000000, {
+      from: accounts[1]
+    })
   })
 
   it('should relay batch', async function () {
-    const transactionResponse = await testToken.transfer(accounts[3], 200000, {
+
+    const tx1 = testToken.transfer(accounts[3], 200000, {
       from: accounts[0],
       gas: innerTransactionGas
     })
 
+    //TODO: removing this sometime causes:
+    // Can't insert key 0xb354ecf032e9e14442be590d9eaee37d2924b67a, it violates the unique constraint"
```

Comment:
This seems to indicate a bug in our storage code, it will be interesting to investigate.